templates/download/desktop/index.html

```diff
@@ -188,19 +194,43 @@ <h2>Ubuntu {{ releases.latest.full_version }}</h2>
             The latest version of the Ubuntu operating system for desktop PCs and laptops, Ubuntu 25.04 comes with nine months of security and maintenance updates, until January 2026.
           </p>
           <hr class="p-rule--muted" />
-          <p class="u-responsive-realign">
-            <a class="p-button--positive"
-               href="/download/desktop/thank-you?version={{ releases.latest.full_version }}&amp;architecture=amd64"
-               onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Download', 'eventAction' : 'Desktop', 'eventLabel' : '{{ releases.latest.short_version }}', 'eventValue' : undefined });">Download
-              {{ releases.latest.full_version }}&nbsp;(AMD64)
-            </a>
-            {% if releases.latest.iso_download_size %}
-              <span class="u-text--muted">{{ releases.latest.iso_download_size }}</span>
-            {% endif %}
-            <script>
-              performance.mark("Download (Desktop) button rendered")
-            </script>
-          </p>
+          <div class="u-responsive-realign">
+            <strong style="font-size: 16px;
+                           padding: 6px 0 0;
+                           margin-right: 32px;
+                           width: 284px">Intel or AMD 64-bit architecture</strong>
+            <p class="u-responsive-realign">
+              <a class="p-button--positive"
+                 href="/download/desktop/thank-you?version={{ releases.latest.full_version }}&amp;architecture=amd64"
+                 onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Download', 'eventAction' : 'Desktop', 'eventLabel' : '{{ releases.latest.short_version }}', 'eventValue' : undefined });">Download
+              </a>
+              {% if releases.latest.iso_download_size %}
+                <span class="u-text--muted">{{ releases.latest.iso_download_size }}</span>
+              {% endif %}
+              <script>
+                performance.mark("Download (Desktop) button rendered")
+              </script>
+            </p>
+          </div>
+          <hr class="p-rule--muted" />
+          <div class="u-responsive-realign">
+            <strong style="font-size: 16px;
+                           padding: 6px 0 0;
+                           margin-right: 32px;
+                           width: 284px">ARM 64-bit architecture</strong>
+            <p class="u-responsive-realign">
+              <a class="p-button"
+                 href="/download/desktop/thank-you?version={{ releases.latest.full_version }}&amp;architecture=arm64"
```

Comment:
Should be this download link, arm versions aren't hosted on releases.ubuntu
```suggestion
                 href="https://cdimage.ubuntu.com/releases/{{ releases.latest.full_version }}/release/ubuntu-{{ releases.latest.full_version }}-desktop-arm64.iso"
```

Reply:
alright. I also didn't see the copy doc in the jira ticket. i'll ask for it next time